app/controllers/banner/BannerDesignsController.scala

```diff
@@ -165,18 +162,43 @@ class BannerDesignsController(
       case _       => None
     }
 
-  def setStatus(rawStatus: String) =
-    authAction.async(circe.json[List[String]]) { request =>
+  private def getAllBannerTests(): ZIO[ZEnv, DynamoError, List[BannerTest]] = {
```

Comment:
I wonder if a better home for this function would be the `DynamoChannelTests` class rather than this controller?

Reply:
Maybe an argument against that is that everything in `DynamoChannelTests` is generic, and this function specifies types of `ChannelTest`. But I'm being a bit picky here!